.kitchen.yml

```diff
@@ -32,6 +32,12 @@ suites:
     verifier:
       inspec_tests:
         - test/smoke/docker
+  - name: rhel-docker
```

Comment:
This is not clear at all and it needs to be added to the README somewhere. This needs to added to the .kitchen.yml.erb file to get Jenkins to test it. .kitchen.yml is used when testing locally.

The Jenkinsfile's Test stage takes the parameters passed in for key pair, security group, and subnet, then generates the .kitchen.yml file used for testing from .kitchen.yml.erb.

Reply:
4517897

> This is not clear at all and it needs to be added to the README somewhere. 

Yes